src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 server.port=9090
+version=@version@
```

Comment:
Same as the other version comment. could benefit from stating what the version is